.github/workflows/bot-comment-notify.yml

```
@@ -0,0 +1,148 @@
+            echo "bot_name=CodeRabbit" >> $GITHUB_OUTPUT
+            echo "bot_icon=🐰" >> $GITHUB_OUTPUT
+            echo "bot_color=#FF6B6B" >> $GITHUB_OUTPUT
+          else
+            echo "bot_name=Unknown Bot" >> $GITHUB_OUTPUT
+            echo "bot_icon=🤷" >> $GITHUB_OUTPUT
+            echo "bot_color=#808080" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Extract comment preview
+        id: comment-preview
+        run: |
```

actionlint (1.7.7)

"github.event.comment.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)